webapp-django/crashstats/crashstats/models.py

```diff
@@ -26,6 +26,7 @@
 import socorro.external.postgresql.gccrashes
 import socorro.external.postgresql.crontabber_state
 import socorro.external.postgresql.adi
+import socorro.external.postgresql.product_build_types
```

Comment:
That would only be possible if `socorro/external/postgresql/__init__.py` imports all those modules.